great_expectations/data_context/store/fixed_length_tuple_store_backend.py

```diff
@@ -60,6 +60,9 @@ def _validate_value(self, value):
                 type(value),
             ))
 
+    def convert_key_to_filepath(self, key):
```

Comment:
I believe this public method should be unimplemented here and only implemented in the File-based subclasses (see below).

Reply:
+1